app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js

```diff
@@ -854,6 +854,26 @@ define([
             },
             $.mage.__('Please enter a value greater than or equal to {0}.')
         ],
+        'validate-range-from': [
```

Comment:
Could you do the same as validate-date-range?
https://github.com/magento-commerce/magento2ce/blob/2.4-develop/lib/web/mage/validation.js#L1055

Reply:
Fixed